tests/test_fields.py

```diff
@@ -237,3 +237,13 @@ def test_update_one_attribute(self):
             hmac_field__hash_of=expected,
         )
         self.assertEqual(updated_instance.first(), instance)
+
+    def test_negative_number(self):
+        """Assert negative number can be saved."""
+        expected = -1
+        instance = EncryptedTextFieldModelFactory.create(
+            integer_pgp_pub_field=expected,
+            integer_pgp_sym_field=expected,
+        )
+        self.assertEqual(instance.integer_pgp_pub_field, expected)
+        self.assertEqual(instance.integer_pgp_sym_field, expected)
```

Comment:
I would prefer to use `subTest` or have two distinct tests here. If there's a problem with `integer_pgp_pub_field` we might not know if the same problem affects `integer_pgp_sym_field`.